docs/src/main/sphinx/security/salesforce.rst

```diff
@@ -19,8 +19,15 @@ basic credentials.  This can also be used to secure the :ref:`Web UI <web-ui-aut
 Salesforce authenticator configuration
 --------------------------------------
 
-Enable Salesforce authentication by creating an
-``etc/password-authenticator.properties`` file on the coordinator:
+Enable password file authentication by setting the :doc:`password authentication
+type <authentication-types>` in ``config.properties``:
```

Comment:
`etc/config.properties` to be consistent, since we're using `etc/` below

Reply:
done